### DISCUSSION TITLE
dock alignment auto hide plz

### DISCUSSION THREAD

**Discussion by alexbog73** [EthanArbuckle/carplay-cast#33] [Ideas]
Is it possible to make the dock alignment hide on its own after a certain time?

**Comment by fahadfardan** [comment]
Yes please please please please please do this. I really need it. The scale with the dock is very bad. I need to hide the dock every time I touch the screen for something to get rid of the scale problem.

**Reply by fahadfardan** [reply]
Also, there is a problem. When the dock appears the scale of the app is not correct until i click on the hide button
